tests/js/query-tests.js

```diff
@@ -77,18 +75,24 @@ function runQuerySuite(suite) {
 
     for (var index in suite.tests) {
         var test = suite.tests[index];
+        var type;
```

Comment:
why this change? these variables are not used outside of this block. no reason to declare them up here.